### PR TITLE
new blog post about Gatsby's pair programming

### PR DESCRIPTION
**Pull request by horacioh**

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit 202fde3d6150969c66619d5ff9fc0b53fcc9e681

https://app.netlify.com/sites/using-postcss-sass/deploys/5b6f78983813f0780158f68d

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit 202fde3d6150969c66619d5ff9fc0b53fcc9e681

https://app.netlify.com/sites/gatsbyjs/deploys/5b6f78963813f0780158f682

**Comment by horacioh** [comment]
@jlengstorf changes done!

File: docs/blog/2018-08-11-gatsby-pair-programming/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+image: images/screenshot.jpg
+author: Horacio Herrera
+tags: ["collaboration","gatsby","community"]
+---
+
+I love pair programming. This is one of the practices we do in our trainings at [ReactJS Academy](https://reactjs.academy?utm_source=social&utm_medium=medium&utm_campaign=horacio-gatsby-post&utm_term=reactjs-academy), and we do it regularly at [LeanJS](https://leanjs.com?utm_source=social&utm_medium=medium&utm_campaign=horacio-gatsby-post&utm_term=leanjs).
+
+A couple of weeks ago, we were planning to rebuild our websites using Gatsby, and then I saw this:
+
+![Jason's Tweet](images/tweet.png)
```
[the remark this answers is not in the document]
Comment (horacioh) [reply docs/blog/2018-08-11-gatsby-pair-programming/index.md]: :+1:

File: docs/blog/2018-08-11-gatsby-pair-programming/index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+### What was the most surprising thing about pair programming?
+
+The most surprising thing to me was Jason’s willingness to spend time with a complete stranger to help me with my questions. I’m grateful that he spent this time with me. Thanks man!
+
+Jason patiently explained the parts I didn’t understand about Gatsby and he also walked me through the project perfectly.
+
+### On a scale of 1–10, how likely are you to recommend it to a friend? (1–10 scale)
+
+10! No doubt. And higher than 10 if you have never done pair programming. Just take a simple task and go! It’s not about the code you will write, but the experience of developing with an expert. It’s just amazing. :)
+
```
[the remark this answers is not in the document]
Comment (horacioh) [reply docs/blog/2018-08-11-gatsby-pair-programming/index.md]: :+1:

**Comment by pieh** [comment]
@horacioh Hey, we didn't notice this in PR review, but we need to update https://github.com/gatsbyjs/gatsby/blob/master/docs/blog/author.yaml with your data - I don't have nice photo of You, so I would appreciate if you could add that data and your photo to `/docs/blog/avatars` directory in follow up PR

**Comment by pieh** [comment]
@horacioh Actually, I'll grab photo from github for now and if you wish, You can swap it later on

**Comment by horacioh** [comment]
@pieh perfect!

thanks!! :tada:

**Pull request merged**